test/discovery.integration.spec.ts

```diff
@@ -296,8 +295,14 @@ it('getExtraRestDescriptions works for google ads', async () => {
   );
 
   // Assert
-  assert.deepStrictEqual(
-    googleAds.map(x => x.restDescription.version),
-    ['v4', 'v5', 'v6', 'v7', 'v8', 'v9', 'v10', 'v11']
-  );
-}).timeout(0); // performs requests to the actual server
+  expect(googleAds.map(x => x.restDescription.version)).toStrictEqual([
+    'v4',
+    'v5',
+    'v6',
+    'v7',
+    'v8',
+    'v9',
+    'v10',
+    'v11',
+  ]);
+}, 0); // performs requests to the actual server
```

Comment:
Turns out setting timeout to 0 doesn't disable timeouts in jest, please set it to 30000 instead, should give it enough time. It took just ~2 seconds for CI so that's why it didn't fail, but it tool just a little bit over the default 5s timeout on my computer (probably Australian internet to blame here)

Reply:
Fixed in https://github.com/Maxim-Mazurok/google-api-typings-generator/commit/a613f11cc0607fc2342e895f556f1d1fc1d05d81